dotnet/samples/Concepts/Memory/VectorSearch_Simple.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Memory.VectorStoreFixtures;
+using Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+using Microsoft.SemanticKernel.Connectors.Qdrant;
+using Microsoft.SemanticKernel.Data;
+using Microsoft.SemanticKernel.Embeddings;
+using Qdrant.Client;
+
+namespace Memory;
+
+/// <summary>
+/// A simple example showing how to ingest data into a vector store and then use vector search to find related records to a given string.
+///
+/// The example shows the following steps:
+/// 1. Create an embedding generator.
+/// 2. Create a Qdrant Vector Store.
+/// 3. Ingest some data into the vector store.
+/// 4. Search the vector store with various text and filtering options.
+///
+/// You need a local instance of Docker running, since the associated fixture will try and start a Qdrant container in the local docker instance to run against.
+/// </summary>
+public class VectorSearch_Simple(ITestOutputHelper output, VectorStoreQdrantContainerFixture qdrantFixture) : BaseTest(output), IClassFixture<VectorStoreQdrantContainerFixture>
+{
+    [Fact]
+    public async Task ExampleAsync()
+    {
+        // Create an embedding generation service.
+        var textEmbeddingGenerationService = new AzureOpenAITextEmbeddingGenerationService(
+                TestConfiguration.AzureOpenAIEmbeddings.DeploymentName,
+                TestConfiguration.AzureOpenAIEmbeddings.Endpoint,
+                TestConfiguration.AzureOpenAIEmbeddings.ApiKey);
+
+        // Initiate the docker container and construct the vector store.
+        await qdrantFixture.ManualInitializeAsync();
+        var vectorStore = new QdrantVectorStore(new QdrantClient("localhost"));
```

Comment:
Should the simplest example just use the volatile memory store?

You could consider adding two Facts and a utility method which takes a vector store instance as a parameter. Fact #1 would use volatile so zero friction and Fact #1 could use Quadrant.

Reply:
Yeah, makes sense.  For now I've just updated it to only use volatile, and I'll add more examples with the samples work item I have.